plugins/chat.py

```diff
@@ -0,0 +1,339 @@
+    else:
+        text = message.input_str
+    if not text:
+        await message.edit(
+            "```Bruh, Without chat name, I can't Join...^_^```", del_in=3)
+        return
+    try:
+        chat = await userge.get_chat(text)
+        await userge.join_chat(text)
+        await userge.send_message(text, f"```Joined {chat.title} Successfully...```")
+    except UsernameNotOccupied:
+        await message.edit("```Username, you entered, is not exist... ```", del_in=3)
+        return
+    except PeerIdInvalid:
+        await message.edit("```Chat id, you entered, is not exist... ```", del_in=3)
+        return
+    else:
+        await message.delete()
+        await asyncio.sleep(2)
+
+
+@userge.on_cmd("leave", about={
+    'header': "Leave Chat",
+    'usage': "{tr}leave\n{tr}leave [chat username | reply to Chat username text]",
+    'examples': [
+        "{tr}leave", "{tr}leave UserGeOt"]},
+    allow_via_bot=True, allow_private=False)
+async def leave_chat(message: Message):
+    """ Leave chat """
+    replied = message.reply_to_message
+    if replied:
+        text = replied.text
+    else:
+        text = message.input_str
+    if not text:
+        text = message.chat.id
+    try:
+        await userge.send_message(text, "```Good bye, Cruel World... :-) ```")
+        await userge.leave_chat(text)
+    except UsernameNotOccupied:
+        await message.edit("```Username, you entered, is not exist... ```", del_in=3)
+        return
+    except PeerIdInvalid:
+        await message.edit("```Chat id, you entered, is not exist...```", del_in=3)
+        return
+    else:
+        await message.delete()
+        await asyncio.sleep(2)
+
+
+@userge.on_cmd("invite", about={
+    'header': "Generate chat Invite link",
+    'usage': "{tr}invite\n{tr}invite [Chat Id | Chat Username]"},
+    allow_channels=False)
+async def invite_link(message: Message):
+    if message.input_str:
+        chat_id = message.input_str
+    else:
+        chat_id = message.chat.id
+    try:
+        chat = await userge.get_chat(chat_id)
+        chat_name = chat.title
+        if chat.type in ['group', 'supergroup', 'private']:
+            link = await userge.export_chat_invite_link(chat_id)
+            await message.edit(
+                "**Invite link Genrated Successfully for\n"
+                f"{chat_name}**\n[Click here to join]({link})",
+                disable_web_page_preview=True)
+        else:
+            await message.edit("```Requirements not met...```")
+    except ChatAdminRequired:
+        if chat.username:
+            await message.edit(
+                f"```You is not admin in @{chat.username}, can't Generate invite link (-_-)```",
+                log=__name__
+            )
+        else:
+            await message.edit(
+                f"```You is not admin in {chat.title}, Can't Generate invite link (-.-)```",
+                log=__name__
+            )
+            return
+    except Exception as e:
+        print(e)
```

Comment:
finally, can you replace this with logger , like 

```suggestion
        LOG.error(e)
```

add `LOG = userge.getLogger(__name__)` on top